### DISCUSSION TITLE
Windows Package Manager 1.8.1911

### DISCUSSION THREAD

**Discussion by ryfu-msft** [microsoft/winget-cli#4627] [Releases]
This is the second stable release of Windows Package Manager 1.8 for Windows 10 (1809+), and Windows 11. 

Experimental features have been disabled in this release.

> **WinGet v1.8 will be the last stable version of WinGet with experimental features disabled. Starting in WinGet v1.9, stable releases will include the ability to enable experimental features unless blocked by Group Policy Objects (GPO).**

## Features
* HTTP Proxy support
* WinGet Configuration Mixed Elevation Mode support via COM
* WinGet Package Icon support via COM
* PowerShell Modules work on ARM64
* Control Characters shouldn't be allowed in user generated content
* Downloading MSStore packages
* [Extracting archives using tar.exe](https://github.com/microsoft/winget-cli/blob/master/doc/Settings.md#archive-extraction-method)
* V2 Index

## What's Changed
* Remove experimental from index v2 by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4611
* Fix for attempted use of capability API before it was introduced by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4623

**Full Changelog**: https://github.com/microsoft/winget-cli/compare/release-v1.8.1791...release-v1.8.1911

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v1.8.1911'>Windows Package Manager 1.8.1911</a>.</em>

**Comment by cjrw2000** [comment]
?

**Reply by cjrw2000** [reply]
What from here

**Comment by kendalldita** [comment]
Where can I find winget.exe?
